src/components/form/date-input/index.tsx

```diff
@@ -115,34 +115,46 @@ export const DateInput = React.forwardRef<HTMLDivElement, DateInputProps>(
 
     return (
       <StyledDayPicker
-        className={cx('dateInput', !isValid && 'error', className)}
+        className={cx('dateInput', (!isValid || error) && 'error', className)}
         theme={theme}
         ref={ref}
         {...wrapperProps}
       >
-        {inputs.map((input, index) => {
-          const inputWidth = input.format.length * 10 + 40;
-
-          const isLastItem = index + 1 !== inputs.length;
-          return (
-            <div className='dateInputItem' key={`text-input-${input.type}`}>
-              <TextInput
-                id={`text-input-${input.type}`}
-                key={`text-input-${input.type}-input`}
-                onChange={(e): void => {
-                  handleChange(e, index, input.type);
-                }}
-                disabled={disabled}
-                inputRef={inputToRef[index]}
-                title={input.title}
-                style={{ width: `${inputWidth}px` }}
-                placeholder={input.placeholder}
-                value={currentDate.split('/')[index]}
-              />
-              {isLastItem ? <Text className='dateInputItemDivider'>{divider}</Text> : null}
-            </div>
-          );
-        })}
+        <div className='dateInputWrapper'>
+          {inputs.map((input, index) => {
+            const inputWidth = input.format.length * 10 + 40;
+
+            const isLastItem = index + 1 !== inputs.length;
+            return (
+              <div className='dateInputItem' key={`text-input-${input.type}`}>
+                <TextInput
+                  id={`text-input-${input.type}`}
+                  key={`text-input-${input.type}-input`}
+                  onChange={(e): void => {
+                    handleChange(e, index, input.type);
+                  }}
+                  disabled={disabled}
+                  inputRef={inputToRef[index]}
+                  title={input.title}
+                  style={{ width: `${inputWidth}px` }}
+                  placeholder={input.placeholder}
+                  value={currentDate.split('/')[index]}
+                />
+                {isLastItem ? <Text className='dateInputItemDivider'>{divider}</Text> : null}
+              </div>
+            );
+          })}
+        </div>
+        {error ? (
+          <Text
+            component='span'
+            size={theme.typography.sizes.xs}
+            color={theme.colors.error}
+            className='inputErrorLabel'
```

Comment:
Same with class name

Reply:
Wouldn't we have to style error label for every component then ?

Reply:
@doichev-kostia ses above^